attributes/default.rb

```diff
@@ -130,8 +130,30 @@
   default['kibana']['kibana_service'] = "http://#{node['kibana']['interface']}:#{node['kibana']['port']}"
 end
 
-#<> The kibana service configuration source
-default['kibana']['service']['source'] = 'upstart.conf.erb'
+# kibana service configurations - defaults to settings for Ubuntu 14.04
+case node['platform']
+when 'centos'
+  if node['platform_version'] < '6.9'
+    default['kibana']['service']['provider'] = Chef::Provider::Service::Init::Redhat
```

Comment:
Chef-client can determine the correct provider directly. Because of your init script you pushed for Upstart's provider, it's not necessary to add this i think (also part in _service.rb)

>The chef-client will determine the correct provider based on configuration data collected by Ohai at the start of the chef-client run. This configuration data is then mapped to a platform and an associated list of providers.